pyqtgraph/functions.py

```diff
@@ -1333,6 +1333,47 @@ def try_fastpath_argb(xp, ain, aout, useRGBA):
     return True
 
 
+def _ndarray_to_qimage(arr, fmt):
```

Comment:
we already have an `arrayToQPath` should we name this with a similar convention?  (I'm just asking, "no" is a perfectly good answer too :) )

Reply:
One issue is whether functions.py is considered public API. ```ndarrayToQImage()``` "looks" too public.
I understand that camelCase is the convention in this library for public facing methods. How about for free standing functions?

Reply:
That's fair.  We don't have clear guidelines on naming conventions, and I think that's a fair assessment to say that cameCase convention is for more public facing methods. 

Unrelated to this PR but we should likely solicit input to form a consistent style guide for the library.  I'm good with leaving this as is 👍🏻